.tekton/source-to-image-rhel9-pull-request.yaml

```diff
@@ -0,0 +1,596 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/openshift/source-to-image?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/cancel-in-progress: "true"
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+      == "master"
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: source-to-image
+    appstudio.openshift.io/component: source-to-image-rhel9
+    pipelines.appstudio.openshift.io/type: build
+  name: source-to-image-rhel9-on-pull-request
```

Comment:
This name is correct. But we need to rename the existing one to `source-to-image-rhel8-on-pull-request`